### PR TITLE
CEM-1232 Warning Component

### PR DESCRIPTION
**Pull request by Ampatte2**
Dashboard has two instances of WarningCard component.

Adding to React-UI library as it may have use cases in future and removes code from Dashboard.

CEM-1232-Warning-Component

### CODE REVIEW

**Review by roggc — APPROVED** [review]
lgtm

File: src/Containers/WarningCard.tsx [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+import { StyledIcon } from 'styled-icons/types';
+import { ExclamationTriangle } from '@styled-icons/fa-solid/ExclamationTriangle';
+import { Trash } from '@styled-icons/fa-solid/Trash';
+import { Card as C, CardProps } from './Card';
+import { Heading } from '../Text';
+import { transition, flex } from '../Utils/Mixins';
+import { Button } from '../Inputs/Button';
+
+export interface IWarningCardProps extends CardProps {
+    action: Function;
```
Comment (ralph-dev) [inline src/Containers/WarningCard.tsx:12]: Always better to define it
() => void
Etc

File: src/Containers/WarningCard.tsx [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+    icon = ExclamationTriangle,
+    iconSize = '30px',
+    buttonIcon = Trash,
+    flexDirection = 'column',
+    children,
+    ...props
+}): React.ReactElement => {
+    const [loading, setLoading] = useState(false);
+    const onClick = async () => {
+        if (!loading && action) {
```
Comment (ralph-dev) [inline src/Containers/WarningCard.tsx:36]: IsMounted?

**Review by ralph-dev — APPROVED** [review]
Lgtm, possible memory leak

**Comment by ralph-dev** [comment]
UseIsMounted hook is much better btw

**Comment by Ampatte2** [comment]
useMounted does not exist yet in React-UI library

**Pull request merged**